### ISSUE TITLE
The search UX exploration

### ISSUE THREAD

**Issue by kdembler** [Joystream/joystream#1699]
During the most recent UI review, we've discovered some discrepancies in assumptions about the search experience in Atlas. The design seems to think of the search as more of an overlay over whatever is displayed on the screen before starting the search, while the current implementation takes a more traditional approach (as seen on Youtube for example) of treating the search as a separate view. This creates differences in expectations - an overlay should be closeable, revealing whatever is behind it in the state it was left. However, a separate view doesn't have this property - it only lets you navigate back to the previews view you've visited, with no guarantee of preserving the state.

For Babylon release, we've decided to postpone any further decision and continue with whatever the current implementation provides. It would be wise to revisit this approach in the future and figure out what's the desired experience.
